pyaerocom/aeroval/setup_classes.py

```diff
@@ -107,7 +108,7 @@ def validate_basedirs(cls, v):
 
     def _check_init_dir(self, loc, assert_exists):
         if assert_exists and not os.path.exists(loc):
```

Comment:
The `and not os.path.exists(loc)` can be skipped. I'm not sure what the assert_exists parameter means.

The `assert_exists` parameter is always true, just remove it? (also in the calls below.)

Reply:
this should perhaps have been addressed outside this PR... this is code in main-dev basically, I added the exist_ok because of the race condition while testing this branch. but also at this point it makes sense to fix it here so I will